blog/views.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.decorators import login_required
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import get_object_or_404, redirect
+from django.views import generic
+
+from blog.models import Post, Commentary
+
+
+class IndexView(generic.ListView):
+    model = Post
+    queryset = Post.objects.all()
```

Comment:
the `queryset` attribute can be omitted in this situation since the `model` attribute already specifies the model for the view.